tests/spatial/affine_test.py

```diff
@@ -0,0 +1,28 @@
+from scipp.spatial import affine_transform, affine_transforms
+import scipp as sc
+
+
+def test_from_affine_matrix():
+    # affine matrix corresponding to a translation by (4, 5, 6)
+    matrix = [[1, 0, 0, 4], [0, 1, 0, 5], [0, 0, 1, 6], [0, 0, 0, 1]]
```

Comment:
This is fine, but for similar things https://numpy.org/doc/stable/reference/generated/numpy.block.html can be useful sometimes (not suggesting to change this here).